graviti/dataframe/column/series.py

```diff
@@ -610,12 +619,62 @@ def _to_post_data(self) -> List[Any]:
 class FileSeries(Series):  # pylint: disable=abstract-method
     """One-dimensional array for file."""
 
-    def __getitem__(self, key: int) -> Optional[FileBase]:
-        scalar = self._data[key]
-        if not scalar.is_valid:
-            return None
+    _data: PagingList  # type: ignore[assignment]
+
+    def __getitem__(self, key: int) -> Any:
+        return self._data[key]
+
+    @classmethod
+    def _from_factory(  # pylint: disable=too-many-arguments
+        cls: Type[_F],
+        factory: LazyFactoryBase,
+        schema: pt.PortexType,
+        object_policy_manager: "ObjectPolicyManager",
+        root: Optional["DataFrame"] = None,
+        name: Tuple[str, ...] = (),
+    ) -> _F:
+        obj: _F = object.__new__(cls)
+        schema_pyobj = schema.to_pyobj()
+        remote_file_type = pt.RemoteFileTypeResgister.SCHEMA_TO_REMOTE_FILE[
+            schema_pyobj["imports"][0]["repo"], schema_pyobj["type"]
```

Comment:
the repo can be get by `schema.package.repo`
the name can be get by `schema.__class__.__name__`